test/integration/filters/test_socket_interface.h

```diff
@@ -27,12 +27,30 @@ class TestIoSocketHandle : public IoSocketHandleImpl {
                      bool socket_v6only = false, absl::optional<int> domain = absl::nullopt)
       : IoSocketHandleImpl(fd, socket_v6only, domain), writev_override_(writev_override_proc) {}
 
+  void initializeFileEvent(Event::Dispatcher& dispatcher, Event::FileReadyCb cb,
```

Comment:
nit: I think this can be `private`.

Reply:
this is an implementation of a public interface method.